### ISSUE TITLE
Can it be used directly on a linear layer?

### ISSUE THREAD

**Issue by sundyCoder** [christiancosgrove/pytorch-spectral-normalization-gan#19]
Can it be used directly on a linear layer? e.g., SpectralNorm(Linear(dim, item)

**Comment by inderjeetnair** [comment]
I guess it can be directly used on linear layers as well. The discriminator architecture in the models.py file itself contains this:
`self.fc = SpectralNorm(nn.Linear(w_g * w_g * 512, 1))`

**Comment by christiancosgrove** [comment]
Yes, that’s right!
